language.txt

```diff
@@ -35,9 +35,10 @@
 # Quoted strings.
 
 "Quoted strings are delimited by double quotes. Backslash denotes escape
-sequences. Legal escape sequences are: \\ \" \x00 \n. \x00 consumes two hex
-digits and emits a byte. Otherwise, any byte before the closing quote,
-including a newline, is emitted as-is."
+sequences. Legal escape sequences are: \\ \" \x00 \000 \n. \x00 consumes two
+hex digits and emits a byte. \000 consumes one to three octal digits and emits
```

Comment:
We should specify that `\000` will reject values that do not fit in a single octet. It feels surprising you can get the parser to emit an extra `0x01` byte in some cases...

Reply:
Done.